docs/notebooks/tutorials/04_Omnigenity_Optimization.ipynb

```diff
@@ -0,0 +1,422 @@
+{
```

Comment:
<p>might want to put closest in quotes to qualify it a bit</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaControl/DESC/pull/535/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>